http/examples/get/server.py

```diff
@@ -0,0 +1,75 @@
+import pyarrow as pa
+from random import randbytes
+from http.server import BaseHTTPRequestHandler, HTTPServer
```

Comment:
@paleolimbot has a Flask-based example here for comparison:
https://github.com/paleolimbot/2023-11-21_arrow-over-http-scratchpad/blob/main/app.py